gc/base/ParallelDispatcher.hpp

```diff
@@ -34,11 +34,13 @@
 #include "modronopt.h"
 #include "modronbase.h"
 
+#include "BaseVirtual.hpp"
 #include "Dispatcher.hpp"
 #include "EnvironmentBase.hpp"
 #include "GCExtensionsBase.hpp"
 
 class MM_EnvironmentBase;
+class MM_Dispatcher;
```

Comment:
Is this declaration actually needed? Since we're subclassing just below, MM_Dispatcher has to be fully defined, right?